apps/editing-toolkit/bin/sync-newspack-blocks.sh

```diff
@@ -134,7 +134,10 @@ echo -n "phpcbf: "
 
 if [ "$PHPCBF_ERRORED" = 1 ] ; then
 	echo '!! There was an error executing phpcbf!'
-	exit 1
+
+	if [ "$MODE" != "npm" ] ; then
```

Comment:
Since the build process does use the `npm` mode, I think this could allow an important issue to slip through in the production build.

That said, I'd still expect phpcbf to be formatting the textdomain correctly most of the time, so it probably won't practically have a huge impact unless the problem is that phpcbf just falls over entirely

Reply:
Good question! What would that help us solve here?

Reply:
`NODE_ENV=production` might be set in the environment, but you might have to double check with `yarn build`

Reply:
> What would that help us solve here?

I imagined we'd add some sort of condition to exit the build process if we're creating a production build _and_ the mode is `npm`